apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/DefaultBlocks.tsx

```diff
@@ -1,52 +1,34 @@
-import { BlocksComponent } from '@ballerine/blocks';
-import { useDefaultBlocksLogic } from '@/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic';
-import { NoBlocks } from '@/lib/blocks/components/NoBlocks/NoBlocks';
-import { ChildDocumentBlocks } from '@/lib/blocks/components/ChildDocumentBlocks/ChildDocumentBlocks';
-import { KycBlock } from '@/lib/blocks/components/KycBlock/KycBlock';
+import { Tabs } from '@/common/components/organisms/Tabs/Tabs';
+import { TabsList } from '@/common/components/organisms/Tabs/Tabs.List';
+import { TabsTrigger } from '@/common/components/organisms/Tabs/Tabs.Trigger';
 import { cells } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { useDefaultBlocksLogic } from '@/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic';
+import { BlocksComponent } from '@ballerine/blocks';
 
 export const DefaultBlocks = () => {
-  const {
-    blocks,
-    kybChildWorkflows,
-    workflowId,
-    parentMachine,
-    onReuploadNeeded,
-    isLoadingReuploadNeeded,
-    kycChildWorkflows,
-    isLoading,
-  } = useDefaultBlocksLogic();
+  const { blocks, tabs, activeTab, setTab } = useDefaultBlocksLogic();
 
   return (
-    <>
+    <div className="relative flex flex-col gap-4">
+      {!tabs.every(tab => tab.hidden) && (
+        <div className="sticky top-0 z-[50] w-full bg-white pb-1">
+          <Tabs value={activeTab?.name} onValueChange={setTab}>
+            <TabsList>
+              {tabs.map(
+                tab =>
+                  !tab.hidden && (
+                    <TabsTrigger key={tab.name} value={tab.name} disabled={tab.disabled}>
+                      {tab.displayName}
+                    </TabsTrigger>
+                  ),
+              )}
+            </TabsList>
+          </Tabs>
+        </div>
+      )}
```

Comment:
Let's filter on `hidden`. This way we could check if `length` is zero and not need the inner-most `!tab.hidden &&` check. Alternatively at the very least use `safeEvery` over `Array.prototype.every`.